docs/contributor_guide/contributing_example_notebooks.rst

```diff
@@ -17,16 +17,60 @@ A good example notebook exhibits the following attributes:
    Avoids abstraction traps.
    See `Selbst et al. (2020) <https://andrewselbst.files.wordpress.com/2019/10/selbst-et-al-fairness-and-abstraction-in-sociotechnical-systems.pdf>`_.
 4. **Substantiated**: Discusses trade-offs and compares alternatives.
-   Describes why using particular Fairlearn functionalities makes sense.
+   Describes why using particular Fairlearn functionalities makes sense if
+   Fairlearn is used.
 5. **For developers**: Speaks the language of developers and data scientists.
    Considers real practitioner needs.
-   Fits within the lifecycle of real practioner work.
+   Fits within the lifecycle of real practitioner work.
    See `Holstein et al (2019) <https://arxiv.org/pdf/1812.05239.pdf>`_,
    `Madaio et al. (2020) <http://www.jennwv.com/papers/checklists.pdf>`_.
 
 Please keep these in mind when creating, discussing, and critiquing examples.
 
-If you'd like to collaborate, please :ref:`reach out <communication>`.
+Process
+^^^^^^^
+
+All current efforts are tracked through items in the corresponding
+`Project Board <https://github.com/fairlearn/fairlearn/projects/3>`_.
+If you'd like to suggest a different kind of project, please
+`open a new issue <https://github.com/fairlearn/fairlearn/issues/new/choose>`_
+if you cannot find a similar one that is already tracked in the project board.
+If you'd like to collaborate with others from the community, please
+:ref:`reach out <communication>` to share your idea.
+
+In the issue, make sure to outline your goals, describe your audience, and
+provide a high-level overview of what you would like to show.
+Note that the examples do not need to contain code.
+In some application contexts it may be preferable not to build a
+technical system, so a textual explanation is perfectly acceptable.
+All contributions should abide by the guidelines outlined above, though.
```

Comment:
I'm glad you're volunteering for this :D if fairlearn starts getting adopted, soon we may not have the resources to answer those questions.

Reply:
Oh definitely! That's a problem I'd love to solve if/when we get to it. Also, we can expand to have more maintainers if we really have such an active community.